### DISCUSSION TITLE
Testing with real repos

### DISCUSSION THREAD

**Discussion by SamWSoftware** [SamWSoftware/serverless-auto-swagger#26] [Ideas]
I was trying to figure out a way to test more effectively with this plugin.

Then I thought, why not just have some sample serverless projects in a "test-repos" folder. We can then run 'sls package' and test the outputed swagger.json file as well as everything else.

This was we don't have to make complicated mocking tools for serverless plugin triggers.

Thoughts? @bfaulk96

**Comment by bfaulk96** [comment]
I love this idea. And it also doubles as examples for people if the README isn't clear enough!

**Reply by SamWSoftware** [reply]
I started on doing this as test for that big PR 17. Once I've tested that, I'll merge 17, create another PR for the test repos and then we should be in a better place for making sure we aren't introducing breaking changes.

**Reply by talyh** [reply]
I was looking for examples and landed in this discussion. Was this ever created?

**Reply by bfaulk96** [reply]
I don't believe it was. I apologize, I've since moved on in my career and do not actively use Serverless, and don't really have the free time outside of work to continue my maintenance on this
